lib/qops/deployment/instances.rb

```diff
@@ -152,7 +152,7 @@ def clean
 
     # Find all instances to be destroyed
     retrieve_instances.each do |instance|
-      next if instance.hostname == 'master'
+      next if instance.hostname == "#{config.hostname_prefix}master"
```

Comment:
Hopefully with some additional changes we can move to fully using tagging. Should be fine either way.